### ISSUE TITLE
Link to create your own instance leads to 404

### ISSUE THREAD

**Issue by okpierre** [syuilo/Misskey#8020]
## 💡 Summary

Link to create your own instance leads to 404

![misskeyreadme404](https://user-images.githubusercontent.com/1679025/143177273-a64735c3-78b2-4728-af4a-07416e4fb8de.png)

## 🙂 Expected Behavior

Redirect to correct install page

## ☹️ Actual Behavior

404 error

## 📝 Steps to Reproduce

1. Go to readme
2. Visit create your own instance link
3. 404 error

## 📌 Environment

Any browser

**Comment by Johann150** [comment]
commit 0134099740631917ff75f48aecba6126f1a5a08c didn't make it but would fix this issue.

Since the link text is in English, maybe we should also better link to the English install guide?

```diff
-[📦 Create your own instance](https://misskey-hub.net/docs/install/install.html)
+[📦 Create your own instance](https://misskey-hub.net/en/docs/install)
```

**Comment by Johann150** [comment]
Fixed in #8017
